src/java/org/apache/nutch/scoring/webgraph/NodeDumper.java

```diff
@@ -86,70 +84,94 @@
    * on the command line, the top urls could be for number of inlinks, for
    * number of outlinks, or for link analysis score.
    */
-  public static class Sorter extends Configured implements
-      Mapper<Text, Node, FloatWritable, Text>,
-      Reducer<FloatWritable, Text, Text, FloatWritable> {
+  public static class Sorter extends Configured {
 
-    private JobConf conf;
-    private boolean inlinks = false;
-    private boolean outlinks = false;
-    private boolean scores = false;
-    private long topn = Long.MAX_VALUE;
+    private static Configuration conf;
+    private static boolean inlinks = false;
+    private static boolean outlinks = false;
+    private static boolean scores = false;
+    private static long topn = Long.MAX_VALUE;
```

Comment:
See comments in Generator etc. regarding static variables shared between mapper and reducer classes.